model/lib/schema.mustache

```diff
@@ -17,4 +17,7 @@ exports.filter = (name, params) => {
   }, {});
 };
 
-exports.validate = jsen(schema);
+exports.getValidator = (name) => {
```

Comment:
Agreed that we don't need to worry too much about breaking API changes yet. This is still in flux. We haven't even released a new version. When we do, it'll be a major.

What I don't want to go unchallenged, though, is that there's no upgrade path.

```
rm -r lib/*
redbeard base myapp
git add -p
git checkout lib
(repeat for other folders)
npm test
git commit -m "Upgraded redbeard"
```

You scaffold out the new files, go line by line to accept upstream changes, keep your local modifications and make edits where appropriate. You commit the result and use your test suite to ensure there have been no regressions.

Any time we change our API in a backwards incompatible way people will need to do work in order to support that, same as any library. The fact that we allow the end user to modify the code so freely means that, if they do so, there's a bit more work to upgrade. Same as if they were maintaining a fork of our library.